grid/example/src/cells/RowEditOverlay.js

```diff
@@ -0,0 +1,215 @@
+import React, { memo, useState } from "react";
+
+const divStyle = {
+    backgroundColor: "#ccc",
+    height: "90px",
+    width: "1365px",
+    marginLeft: "-1370px",
+    marginTop: "-5px"
+}
+
+const segmetEditStyle = {
+    marginLeft: "240px",
+    marginTop: "-60px"
+}
+
+const weightPercentageStyle = {
+    width: "25%",
+    marginLeft: "410px",
+    marginTop: "-20px",
+}
+
+const weightValueStyle = {
+    width: "25%",
+    marginLeft: "410px",
+    marginTop: "6px",
+}
+const srStyle = {
+    marginLeft: "410px",
+    marginTop: "4px"
+}
+
+const editRemarksStyle = {
+    marginTop: "-75px",
+    marginLeft: "700px"
+}
+
+const buttonDivStyle = {
+    width: "15%",
+    marginTop: "-80px",
+    marginLeft: "1244px"
+}
+const RowEditOverLay = memo((props) => {
+    const { row, airportCodeList, updateCellData } = props;
```

Comment:
As part of code optimization, we can use 
const {flight,segment, weight, sr, remarks} = row.original, and then use just flight,segment, weight, sr, remarks in below codes.

This optimization technique is called object destructuring.

Apply same kind of code optimizations in anywhere applicable below

Reply:
object destructuring implemented